README.md

```diff
@@ -17,6 +17,6 @@ This script depends on having firefox downloaded
 
 You must be on VPN in order to run this script.
 
-`bundle exec rspec`
+`TECHMD_HMAC={hash-value} bundle exec rspec`
```

Comment:
This is unnecessary, we can just look for the techMetadata in argo:  for example the "Technical Metadata" section on this page: https://argo-stage.stanford.edu/view/druid:jw234dx5637?beta=true

Reply:
Happy to make the change but was following the suggestions in the ticket

Reply:
The TechMd is not an external API, so I don't think we want our integration tests to be calling it.